config/config.go

```diff
@@ -12,48 +12,51 @@ import (
 var cfg *config
 
 const (
-	EnvEnvironment               = "ENVIRONMENT"
-	EnvDebug                     = "DEBUG"
-	EnvIgnoreDaemonSets          = "IGNORE_DAEMON_SETS"
-	EnvDeleteLocalData           = "DELETE_LOCAL_DATA" // Deprecated: in favor of DeleteEmptyDirData (DELETE_EMPTY_DIR_DATA)
-	EnvDeleteEmptyDirData        = "DELETE_EMPTY_DIR_DATA"
-	EnvClusterName               = "CLUSTER_NAME"
-	EnvAutodiscoveryTags         = "AUTODISCOVERY_TAGS"
-	EnvAutoScalingGroupNames     = "AUTO_SCALING_GROUP_NAMES"
-	EnvAwsRegion                 = "AWS_REGION"
-	EnvExecutionInterval         = "EXECUTION_INTERVAL"
-	EnvExecutionTimeout          = "EXECUTION_TIMEOUT"
-	EnvPodTerminationGracePeriod = "POD_TERMINATION_GRACE_PERIOD"
-	EnvMetrics                   = "METRICS"
-	EnvMetricsPort               = "METRICS_PORT"
-	EnvSlowMode                  = "SLOW_MODE"
-	EnvEagerCordoning            = "EAGER_CORDONING"
+	EnvEnvironment                      = "ENVIRONMENT"
+	EnvDebug                            = "DEBUG"
+	EnvIgnoreDaemonSets                 = "IGNORE_DAEMON_SETS"
+	EnvDeleteLocalData                  = "DELETE_LOCAL_DATA" // Deprecated: in favor of DeleteEmptyDirData (DELETE_EMPTY_DIR_DATA)
+	EnvDeleteEmptyDirData               = "DELETE_EMPTY_DIR_DATA"
+	EnvClusterName                      = "CLUSTER_NAME"
+	EnvAutodiscoveryTags                = "AUTODISCOVERY_TAGS"
+	EnvAutoScalingGroupNames            = "AUTO_SCALING_GROUP_NAMES"
+	EnvAwsRegion                        = "AWS_REGION"
+	EnvExecutionInterval                = "EXECUTION_INTERVAL"
+	EnvExecutionTimeout                 = "EXECUTION_TIMEOUT"
+	EnvPodTerminationGracePeriod        = "POD_TERMINATION_GRACE_PERIOD"
+	EnvMetrics                          = "METRICS"
+	EnvMetricsPort                      = "METRICS_PORT"
+	EnvSlowMode                         = "SLOW_MODE"
+	EnvEagerCordoning                   = "EAGER_CORDONING"
+	EnvExcludeFromExternalLoadBalancers = "EXLUDE_FROM_EXTERNAL_LOAD_BALANCERS"
```

Comment:
I just noticed you made a typo in the environment variable 😂

Reply:
Resolved by 3b8647b

Reply:
Oh damn it :D thanks for fixing this!